### ISSUE TITLE
How are you planning to gather the data to train your model?

### ISSUE THREAD

**Issue by VishalG94** [SJSUFall2019-CMPE272/Virtual-Yoga-Trainer#4]
Since data is key to effectively train any ML model, how are you planning to gather the required data for this project. Are you using any readily available dataset online, or would you gather your own data and label them accordingly.

**Comment by harshrajm** [comment]
We are using pre-trained Posenet model from Tensorflow.js  to track the pose of the user.
